Makefile

```diff
@@ -12,7 +12,11 @@ CMD_GITHUB ?= gh
 # environment:
 ARCH ?= $(shell uname -m)
 KERN_RELEASE ?= $(shell uname -r)
-KERN_SRC ?= /lib/modules/$(KERN_RELEASE)/build
+KERN_BLD_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/build)
+KERN_SRC_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/source)
```

Comment:
why did we keep both KERN_SRC and KERN_SRC_PATH? don't they mean the same thing?

Reply:
They don't. See my above comment for the meaning of KERN_SRC.
KERN_SRC_PATH is the path where the includes that are not generated from the build are located.
KERN_BUILD_PATH is the path where the includes that are generated from the build are located.

On some distros, these two will be on the same directory, while on others, they are splitted